MailResources/Localizable/it.lproj/Localizable.strings

```diff
@@ -1035,7 +1032,7 @@
 "recentSearchesTitle" = "Ricerche recenti";
 
 /* loco:62baabfc07e6a001ca19e535 */
-"refreshTokenError" = "Sei stato disconnesso a causa di un problema. Riconnettiti";
+"refreshTokenError" = "Siete stati disconnessi";
```

Comment:
Can you fix the string on Loco and re-import it?
The correct string should be "Sei stato disconnesso"